core/src/main/scala/org/apache/spark/h2o/converters/SupportedDataset.scala

```diff
@@ -41,15 +41,15 @@ object SupportedDataset {
     val typeTag = ttag
     override def toH2OFrame(sc: SQLContext, frameKeyName: Option[String]): H2OFrame = {
       val tpe = ttag.tpe
-      val constructorSymbol = tpe.declaration(nme.CONSTRUCTOR)
+      val constructorSymbol = tpe.declaration(nme.CONSTRUCTOR) // To be compatible with Scala 2.10
```

Comment:
Because we need to compile for Spark 2.10 and 2.11. I tried to switch to new Spark 2.11 API but it is not compatible with 2.10